avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -954,6 +954,10 @@ private void addAvroRecordField(
       defaultVal = fieldSchema.defaultValue();
       if (defaultVal instanceof Byte) {
         defaultVal = ((Byte) defaultVal).intValue();
+      } else if (defaultVal instanceof Short) {
+        defaultVal = ((Short) defaultVal).intValue();
+      } else if (defaultVal instanceof ByteBuffer) {
+        defaultVal = ((ByteBuffer) defaultVal).array();
```

Comment:
it took me a bit to understand why we are cherry-picking some types here, and not accounting for all primitive types (int, long ... etc). do you think a comment would be helpful in describing the selection rationale here (specifically, that we are handling cases that the `org.apache.avro.util.internal.JacksonUtils#toJson` does not)?

Reply:
ack.